05_iigo/index.tex

```diff
@@ -347,52 +348,69 @@ \section{Accountability Cycle}
+\item President deciding the allocation of resources from the common pool.
+\item Speaker calling a vote and calculating the winner.
+\item Judge inspecting an island's actions.
+\item Judge inspecting an island's action history retroactively.
 \item Declaring (e.g. \textit{announcing} the result of a vote).
-\item Setting the amount of \emph{taxation} for each island.
-\item Deciding the allocation distribution for each island.
-\item Inspecting an island's action history.
+\item Holding an election.
 \item Monitoring a role.
 \end{itemize}
 
 IIGO has been designed to act in the common good. Therefore IIGO-related costs will be directly withdrawn from the common pool. Since the common pool is considered communal property of the archipelago, there are rules in place to limit how much each role is allowed to spend in order to perform its own institutional-power-enabled actions. This is the reason for defining the \emph{budget} and keeping it separate for each of the three IIGO roles.
 
-
-As a role performs institutional-enabled-power actions, the cost associated with this action is subtracted from the role's budget. A budget of zero means that the role does not have the \emph{power} to perform any of its institutional-power-enabled actions. The removal of the budget rule from the rules in play means the role is allowed to perform as many such actions as it would like (as long as those actions are not governed by other rules). 
+\begin{rule_IIGO} \label{rule:budget}
+    %Each role has the \emph{obligation} to pay the salary of amount $S$ to another if and only if the amount paid $S'$ is equal to $S$.
+    Each role has the \emph{permission} to act on an institutional-power action with an associated cost if the budget would not become negative as a result of performing the action.
+ \end{rule_IIGO}
 
 
 
-%As a role performs an endogenous-cost action, the corresponding cost associated with this action is subtracted from the role's budget. Thus, if as a result of an endogenous-cost action a role will be left with negative budget, it will have gone over the budget limit and will have broken the budget rule. A budget of zero means that the role is not allowed to perform any of its institutional-power-enabled actions associated an endogenous-cost. The removal of the budget rule from the rules in play means the role is allowed to perform as many such actions as it pleases (as long as those actions are not governed by other rules). 
+When a role acts on an institutional-power-enabled action with a cost, the cost associated with this action is subtracted from the role's \emph{budget}. If Rule~\ref{rule:budget} is in play, a budget of zero or less means that the role does not have the \emph{permission} to perform any of its institutional-power actions. The removal of Rule~\ref{rule:budget} from the rules in play means the role is permitted to perform as many such actions as it would like (as long as those actions are not governed by other rules).
```

Comment:
```suggestion
When a role acts on an institutionally-enabled-power action that comes with a cost, the cost associated with this action is subtracted from the role's \emph{budget}. If Rule~\ref{rule:budget} is in play, a budget of zero or less means that the role does not have the \emph{permission} to perform any of its institutional-power actions. The removal of Rule~\ref{rule:budget} from the rules in play means the role is permitted to perform as many such actions as it would like (as long as those actions are not governed by other rules).
```

Reply:
An action does not come with a cost. The action has a cost

Reply:
ok